bundle/stable/eclipse-che-preview-openshift/manifests/org_v1_che_crd.yaml

```diff
@@ -556,8 +556,7 @@ spec:
                     cheFlavor:
                       description: Deprecated. The value of this flag is ignored.
                         Specifies a variation of the installation. The options are
-                        `che` for upstream Che installations, or `codeready` for link:https://developers.redhat.com/products/codeready-workspaces/overview[CodeReady
-                        Workspaces] installation. Override the default value only
+                        `che` for upstream Che installations, or `devspaces` for link:https://developers.redhat.com/products/codeready-workspaces/overview[Red Hat OpenShift Dev Spaces] installation. Override the default value only
```

Comment:
I don't know why this didn't wrap like before but if it's autogenerated I suppose there's nothing to be done.